misc/hv_prebuild/vm_cfg_checks.c

```diff
@@ -140,9 +140,21 @@ bool sanitize_vm_config(void)
 			}
 			break;
 		case SOS_VM:
-			if ((vm_config->severity != (uint8_t)SEVERITY_SOS) || ((vm_config->guest_flags & GUEST_FLAG_LAPIC_PASSTHROUGH) != 0U)) {
+			if (vm_config->severity != (uint8_t)SEVERITY_SOS) {
```

Comment:
@ZideChen0 , yes we are switching to XML-based config tool, and feature/FLAG check could  be handled by XML schema, just like you did for SCHEDULER:  <xs:assert test="not (NVMX_ENABLED = 'y' and not (SCHEDULER = 'SCHED_NOOP'))"/> So could you move this sanity check into schema from vm_cfg_checks.c ?

Reply:
Refined sanity check to XML schema, looks good to me now, thanks.